internal/controller/modelregistry_controller_test.go

```diff
@@ -40,6 +40,8 @@ import (
 	"github.com/opendatahub-io/model-registry-operator/api/v1alpha1"
 )
 
+const DescriptionPrefix = "Test Registry "
```

Comment:
I understand this will create a test CR description as `"Test Registry {name}"`, hence the trailing space, which is later used for assertion